docs/rst/manual/theory/ensemble_based_methods.rst

```diff
@@ -397,9 +397,97 @@ Iterative Ensemble Smoother - Ensemble subspace version
 The algorithm implemented is described in the article [Efficient Implementation of an Iterative Ensemble Smoother for Data Assimilation and Reservoir History Matching]( https://www.frontiersin.org/articles/10.3389/fams.2019.00047/full ).
 
 
+Kalman Posterior Properties
+---------------------------
 
+The updating from the prior :math:`p(\psi)=N\left(\mu_\psi,\Sigma_\psi\right)` 
+to the posterior :math:`p(\psi|d)=N\left(\mu_{\psi|d},\Sigma_{\psi|d}\right)`, assimilating measurements :math:`d`, 
+linear in :math:`\psi` is performed by the Kalman methods by employing the following equations
 
+.. math::
+   \begin{align}
+   \mu_{\psi|d} &= \mu_{\psi} + K(d-M\mu_{\psi}),\\
+   \Sigma_{\psi|d} &= (I-KM) \Sigma_{\psi}
+   \end{align}
+
+where
+
+.. math::
+   \begin{align}
+   K = \Sigma_{\psi}M^\top (M\Sigma_{\psi}M^\top + \Sigma_{d})^{-1}
+   \end{align}
+
+is called tha Kalman gain, and :math:`M` is the linear measurement matrix, so that
```

Comment:
Can a matrix be non-linear?

Reply:
It cannot! How about saying `linear measurement operator (i.e., a matrix)` (I think this is almost exactly what Evensen says)?

Reply:
That's nice